internal_docs/how-to-do-the-monthly-update.md

```diff
@@ -0,0 +1,32 @@
+# Writing the monthly update
+
+For us as the Flux project, it's important we explain well what has
+been going on in our space, in all the aspects: development, community,
+documentation, ecosystem, events and everything else.
+
+We have been informing our community on <https://fluxcd.io/blog/> and
+other channels since December 2020.
+
+## Preparing the update
+
+To start a new entry, you simply run
+
+```cli
+./hack/new-update-post.py
```

Comment:
Does this needs a specific Python version what about any pip packages?

Reply:
It doesn't need any, or anything special.

For running the website locally, you do though: https://github.com/fluxcd/website/#readme